frontend/src/components/Spiner.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Spin } from 'antd';
+import styled from "styled-components";
+import { LoadingOutlined } from '@ant-design/icons';
+const antIcon = <LoadingOutlined style={{ fontSize: 24 }} spin />;
+
+const SpinerStyle = styled.div`
```

Comment:
This will fail if we want to add spinner inside a Modal or a section since it uses position fixed.

Reply:
Let's check if we can use some components from antd solving our purpose for div. Maybe Grid? https://ant.design/components/grid/

Reply:
@jyash97 have checked in the modal ya its working fine screenshot

![Screenshot from 2021-07-18 23-54-38](https://user-images.githubusercontent.com/8166149/126078281-57e688be-cb59-4900-8e31-90f7ca92ab67.png)

or did i need to remove the position property itself or i hv to use relative ?

Reply:
@nidhi-tandon instead **_styled.div_** r u referring **_styles.Row_** nidhini? is it correct ?